config/parser/node.go

```diff
@@ -65,6 +65,7 @@ type Hedera struct {
 	Network        string            `yaml:"network"`
 	Rpc            map[string]string `yaml:"rpc"`
 	StartTimestamp int64             `yaml:"start_timestamp"`
+	MaxRetry       int               `yaml:"max_retry" default:"5"`
```

Comment:
https://github.com/hashgraph/hedera-sdk-go/blob/main/transaction.go#L74 - we actually downgrade it, either switch to 10 or more